docs/JSON-RPC.md

```diff
@@ -471,3 +488,42 @@ Parameters:
 | params | object | No parameters (empty object). |
 
 
+### jamulusclient/recorderState
+
+Emitted when the client is connected to a server who's recorder state changes.
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.state | number | The recorder state |
+
+
+### jamulusclient/serverInfoReceived
+
+Emitted when a server info is received.
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.address | string | The server socket address |
+| params.pingtime | number | The round-trip ping time in ms |
+| params.numClients | number | The quantity of clients connected to the server |
+
+
+### jamulusclient/serverListReceived
+
+Emitted when the server list is received.
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.servers | array | The server list. |
+| params.servers[*].address | string | Socket address (ip_address:port) |
+| params.servers[*].name | string | Server name |
+| params.servers[*].country | string | Server country |
```

Comment:
Also include `countryId` for consistency with `jamulusclient/clientListReceived` suggestion above.

Reply:
```suggestion
| params.servers[*].countryId | number | Server country ID (see QLocale::Country). |
| params.servers[*].country | string | Server country |
```